app/analytics.go

```diff
@@ -46,7 +46,6 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 
 		openChan := a.Srv.Store.Channel().AnalyticsTypeCount(teamId, model.CHANNEL_OPEN)
 		privateChan := a.Srv.Store.Channel().AnalyticsTypeCount(teamId, model.CHANNEL_PRIVATE)
-		teamChan := a.Srv.Store.Team().AnalyticsTeamCount()
```

Comment:
This wasn't about initialization or declaration, it was about running things concurrently, I would expect here to follow the "explicit async" usage here (you can see it in the example PR).

Previously we run the 3 selects in the database concurrently, with your change, we run the first 2 concurrently and after that, the third one.

Reply:
Ah ok, I think I missed that. I was under the impression that all those calls were going to be switched over to sync. My bad. I'll fix that up.

Reply:
Yes, is kinda confusing because the proposed changes is for "sync by default", but is exactly that, by default, when you don't need async behavior, you just the store like any other golang function, and when you need run it asynchronously, you now need to do it explicitly, like in this case. :)